### PR TITLE
RHIROS-458 fixed default filtering to name

### PR DESCRIPTION
**Pull request by PreetiW**
Details:
Fixed the default filtering to name
Jira: https://issues.redhat.com/browse/RHIROS-458

Screenshot:

image

### CODE REVIEW

File: src/Routes/RosPage/RosPage.js [hunk 1]
```diff
@@ -240,6 +240,7 @@ class RosPage extends React.Component {
                                     {
                                         label: SFObject.label,
                                         type: SFObject.type,
+                                        value: `checkbox-state`,
```
Comment (PreetiW) [inline src/Routes/RosPage/RosPage.js:243]: @karelhala @kgaikwad Based on the code here https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/ConditionalFilter/ConditionalFilter.js#L133 and https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/ConditionalFilter/ConditionalFilter.js#L62 and referring others frontend repo I made this change.
Let me know if you have suggestion on better approach for same.

Comment (karelhala) [reply src/Routes/RosPage/RosPage.js:243]: Try removing the value from `filterValues` and add it here
```suggestion
                                        value: this.state.stateFilterValue,
```

**Review by karelhala — COMMENTED** [review]
The value should be real value and not just static constant.

File: src/Routes/RosPage/RosPage.js (hunk 1 above)
Comment (PreetiW) [reply src/Routes/RosPage/RosPage.js:243]: @karelhala I tried this but with this change as soon as I select the state filter from the list the dropdown changes to Name again. Here is the screenshot for your reference
image

Comment (karelhala) [reply src/Routes/RosPage/RosPage.js:243]: Aaah yeah! I thought you were setting the value for the whole conditional filter. This is the right approach and works as should.

**Review by karelhala — APPROVED** [review]

**Review by kgaikwad — APPROVED** [review]
Working fine!

**Comment by kgaikwad** [comment]
Thank you @PreetiW and @karelhala for review!

**Pull request merged**